src/lib/render/RenderStatistic.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { ConfigProvider, type Config } from "../../contexts/config-context.tsx";
+import ReactDOM from "react-dom/client";
+import { Theme } from "../../components/Theme";
+import { Statistic } from "../../components/Statistic/Statistic";
+import { useEventBusForStatistic } from "../../hooks";
+
+export function renderStatistic(element: HTMLElement, config: Config) {
+  const StatisticTab: React.FC<Config> = (config) => {
+    const { backFromStatistic } = useEventBusForStatistic();
+
+    return (
+      <ConfigProvider config={config}>
+        <Theme>
+          <Statistic backFromStatistic={backFromStatistic} />
```

Comment:
looks like we're heading in the right direction :)

I've been putting large components that use/call the event bus in in the `src/features` directory. 

Then adding renderer for it in `lib` so it can be exported like a component :)

Reply:
fixed